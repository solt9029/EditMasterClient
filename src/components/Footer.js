import React from 'react';
import styled from 'styled-components';
import Container from '../styled/Container';

const StyledContainer = styled(Container)`
  padding: 10px;
  background: linear-gradient(
    45deg,
    rgba(20, 160, 140, 0.9),
    rgba(70, 150, 180, 0.8)
  );
  color: #fff;
`;

const Footer = () => {
  return (
    <StyledContainer fluid center={1} top={50}>
      <small>Copyright Â© Kenshi Shiode. All Rights Reserved.</small>
    </StyledContainer>
  );
};

export default Footer;
