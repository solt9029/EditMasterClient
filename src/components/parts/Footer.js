import React, { Component } from 'react';
import { Container } from 'reactstrap';
import styled from 'styled-components';

const StyledContainer = styled(Container)`
  text-align: center;
  padding: 10px;
  background: linear-gradient(
    45deg,
    rgba(20, 160, 140, 0.9),
    rgba(70, 150, 180, 0.8)
  );
  color: #fff;
`;

export default class Footer extends Component {
  render() {
    return (
      <StyledContainer fluid>
        <small>Copyright Â© Kenshi Shiode. All Rights Reserved.</small>
      </StyledContainer>
    );
  }
}
