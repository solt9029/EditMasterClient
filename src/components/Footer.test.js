import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './Footer';

describe('Footer', () => {
  test('snapshot', () => {
    const result = renderer.create(<Footer />).toJSON;
    expect(result).toMatchSnapshot();
  });
});
