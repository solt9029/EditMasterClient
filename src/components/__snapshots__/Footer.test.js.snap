// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot 1`] = `[Function]`;
